tests/png-suite/tests/transparency.t/run.t

```diff
@@ -0,0 +1,30 @@
+  $ ../test.sh *.png
+  Testing tbbn0g04.png
+  Corrupted image: cannot write greyscale image with alpha to PNG with bitdepth of 4.
+  
+  Testing tbbn2c16.png
+  0
+  Testing tbbn3p08.png
+  0
+  Testing tbgn2c16.png
+  0
+  Testing tbgn3p08.png
+  0
+  Testing tbrn2c08.png
+  453
```

Comment:
I'm not sure why there is a difference here, but my guess is that this again has to do with `tRNS` handling. I suspect the issue is again with the PNG output, but I did not confirm.